### PR TITLE
Core 948 lending home update backgrounds

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- background for every section updated
- width fixed as /lend/filter page specs

### CODE REVIEW

File: src/pages/LoanFinding/LoanFinding.vue [hunk 1]
```diff
@@ -1,16 +1,19 @@
 <template>
-	<www-page class="tw-bg-secondary" style="height: auto;">
-		<div class="tw-max-w-5xl tw-mx-auto tw-p-2 lg:tw-pt-4">
-			<h3 class="tw-text-h3 tw-text-primary">
-				Welcome back, <span class="tw-text-action fs-mask data-hj-suppress">{{ firstName }}</span>
-			</h3>
+	<www-page class="tw-bg-white" style="height: auto;">
+		<div class="tw-w-full">
+			<!-- eslint-disable-next-line max-len -->
+			<div class="tw-mx-auto tw-p-2 lg:tw-pt-4 tw-px-2.5 md:tw-px-4 lg:tw-px-8 tw-bg-white" style="max-width: 1200px;">
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:5]: Can this max width just be in this component in a wrapper div (as opposed to on each component)?

File: src/components/LoanFinding/PartnerSpotlightSection.vue [hunk 2]
```diff
@@ -1,13 +1,15 @@
 <template>
-	<div>
-		<m-f-i-hero />
+	<div class="tw-w-full tw-bg-secondary">
```
Comment (dyersituations) [inline src/components/LoanFinding/PartnerSpotlightSection.vue:2]: I was hoping we could pull these BGs from contentful, but I see that this actually isn't a contentful-driven page.

File: src/pages/LoanFinding/LoanFinding.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:5]: No b/c every section need to be full width to show the bg correctly and the content inside within 1200px width

Comment (dyersituations) [reply src/pages/LoanFinding/LoanFinding.vue:5]: Makes sense, thanks for considering.

File: src/components/LoanFinding/PartnerSpotlightSection.vue (hunk 2 above)
Comment (roger-in-kiva) [reply src/components/LoanFinding/PartnerSpotlightSection.vue:2]: yeah, we need to add some extra work to move this to contentful

File: src/pages/LoanFinding/LoanFinding.vue [hunk 3]
```diff
@@ -1,16 +1,19 @@
 <template>
-	<www-page class="tw-bg-secondary" style="height: auto;">
-		<div class="tw-max-w-5xl tw-mx-auto tw-p-2 lg:tw-pt-4">
-			<h3 class="tw-text-h3 tw-text-primary">
-				Welcome back, <span class="tw-text-action fs-mask data-hj-suppress">{{ firstName }}</span>
-			</h3>
+	<www-page class="tw-bg-white" style="height: auto;">
```
Comment (mcstover) [inline src/pages/LoanFinding/LoanFinding.vue:2]: @roger-in-kiva can you use the `main-class` prop here?

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue]: yeah, done

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue]: ooh I see is a vue prop in the component and not a class

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue]: added now

File: src/pages/LoanFinding/LoanFinding.vue (hunk 1 above)
Comment (mcstover) [reply src/pages/LoanFinding/LoanFinding.vue:5]: Having `tw-bg-white` on line 5 here is redundant now that the page bg is white.

File: src/pages/LoanFinding/LoanFinding.vue (hunk 3 above)
Comment (mcstover) [reply src/pages/LoanFinding/LoanFinding.vue]: Thanks!

File: src/pages/LoanFinding/LoanFinding.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue]: right, removed

**Review by dyersituations — APPROVED** [review]
Ideally in the future we add a "sized wrapper" shared component. For example, with this implementation, you'd have to change "1200" in a bunch of places. Ideally there's a simple wrapper that owns that size and is used everywhere.

**Pull request merged**